pyproject.toml

```diff
@@ -50,7 +49,7 @@ dev = [
 
 
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools", "setuptools-scm"]
```

Comment:
I should have done this a week or two ago when I did the socorro-siggen update. This allows us to ditch fiddling with `MANFIEST.in`.

https://pypi.org/project/setuptools-scm/